src/menu/menu.scss

```diff
@@ -94,6 +119,15 @@
       }
     }
   }
+
+  > .iui-divider {
+    height: 1px;
+    width: calc(100% - #{$iui-l});
```

Comment:
You don't need `#{}`

Reply:
I think it is required, notice the width of the dividing line.  With `#{ }`:
![Screen Shot 2021-06-28 at 11 06 41 AM](https://user-images.githubusercontent.com/849817/123659653-ecd65080-d800-11eb-8129-c0bb0fc3f193.png)

Without `#{ }`:
![Screen Shot 2021-06-28 at 11 07 08 AM](https://user-images.githubusercontent.com/849817/123659726-fc559980-d800-11eb-9481-5795a480fe22.png)

Reply:
Oh right, because it's inside `calc`